notebooks/RAG_evaluation.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>This would be a good place to show a few lines from the dataset, or give a link to your pregenerated dataset, so that a reader could inspect it and understand better what you've built.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done! And I add a disclaimer on "now that we've seen how to generate this on a few examples, let's load a big pre-generated dataset"</p>